data/human/transport missions.txt

```diff
@@ -3594,6 +3595,747 @@ mission "Hauler VI cargo prototype 3"
+	source "Sunracer"
+	destination "Stormhold"
+	cargo "Factory Equipment" 79
+	blocked "You are going to need more cargo space for the coming mission."
+	clearance `Beto manages to convince the pirate spaceport to let you land "free of charge."`
+	waypoint "Zaurak"
+	on enter "Zaurak"
+		conversation
+			`You enter the <system> and spot the "Union Strong" on your scanners. They must see you too because they detonate a string of charges placed around their hull and engines, simulating a pirate attack. Hopefully the insurance company won't look too deeply.`
+	to offer
+		has "Freedom Uprising 2: done"
+	passengers 3
+	on offer
+		conversation
+			`You meet up with everyone at what must be the seediest dive bar in the spaceport. "That didn't go exactly as expected," informs Beto. "The unions are unable to help us directly so we need to change the plan a bit. The good news is there are three small warships waiting for us in space, a great addition to our humble 'Freedom Fighters' defense force. Also, they are going to wire us one million credits to buy arms. But getting the factory equipment is going to be a little more complicated."`
+			`	"They can't give us the equipment outright," chimes in Bunker while nursing a beer. "But the union has put in an order for exactly what is needed for the factory on Freedom. They also arranged for union guys to do the transport. Unfortunately their ship, the 'Union Strong', will be 'attacked' in transit. Luckily for them you will be there just in time to save them. And by save them I mean board their ship and offload the equipment. After that we go to <destination> to do some interviews and pick up the rest of what we need before going to Freedom. It's a rough system but we should be safe once we land so hail the spaceport and Beto should be able to get us permission to land without paying protection."`
+			`	"We are going to go though a lot of empty space soon so my contacts will have a Ramscoop waiting for us on Stormhold, but also consider installing more Fuel Pods here because they won't be available along the way."`
+
+			choice
+				`	"Sounds like a plan. Let's do it!"`
+					accept
+				`	"I thought you all were against piracy?"`
+			`	"We aren't against piracy per se," says Beto. "There is nothing wrong about stealing from your oppressor. But many pirates are individualists who will rob, kill or enslave anyone for their own self gain. That we can not abide by."`
+
+			choice
+				`	"This is nuts. I can't steal from the Syndicate."`
+					goto steal
+				`	"OK. Let's do this."`
+					accept
+			label steal
+			`	"I understand," says Ale. "It's a big risk and a lot to ask. We will be in touch if there is other more tame work available."`
+				decline
+
+	on visit dialog `You either forgot to board the 'Union Strong' or you left your escorts behind.`
+
+	on complete
+		payment 1000000
+		outfit "Ramscoop" +1
```

Comment:
You don't need a plus sign for `outfit`. Any number that isn't negative is already adding.
```suggestion
		outfit "Ramscoop" 1
```